pkg/client/image/image.go

```diff
@@ -35,6 +39,37 @@ func (c *Client) Delete(id string) error {
 	return c.client.Delete(id, c.instanceID)
 }
 
+//func ImportImage imports image from S3 Instance
+func (c *Client) ImportImage(instanceID, imageName, s3Filename, region, accessKey, secretKey, bucketName, osType, diskType string) (*models.Image, error) {
+	var source = "url"
+	var body = models.CreateImage{
+		ImageName:     imageName,
+		ImageFilename: s3Filename,
+		Region:        region,
+		AccessKey:     accessKey,
+		SecretKey:     secretKey,
+		BucketName:    bucketName,
+		OsType:        osType,
+		DiskType:      diskType,
+		Source:        &source,
+	}
+
+	params := p_cloud_images.NewPcloudCloudinstancesImagesPostParamsWithTimeout(pkg.TIMEOUT).WithCloudInstanceID(instanceID).WithBody(&body)
+	resp, err, _ := c.session.Power.PCloudImages.PcloudCloudinstancesImagesPost(params, ibmpisession.NewAuth(c.session, instanceID))
+
+	if err.Payload.State == "queued" {
+		klog.Infof("Post is successful %s", *err.Payload.ImageID)
+
+	}
+
+	if resp != nil {
+		klog.Infof("Failed to initiate the copy job ")
```

Comment:
this message does not make sense to me, can be enhanced to make it more sense to this function and need a error return from here